docs/openapi.yaml

```diff
@@ -612,6 +655,134 @@ components:
             type: string
         title:
           type: string
+    ImportCandidate:
```

Comment:
Is it possible to extend ImportCandidate from publication like you do in typescript in open-api? Just wondering

Reply:
i will test "allOf" property in openapi scheme, changing it back if it does not work